dags/imagegen_devx/jax_stable_stack_tpu_e2e.py

```diff
@@ -27,7 +27,7 @@
 
 
 with models.DAG(
-    dag_id="jax_stable_stack_e2e",
+    dag_id="jax_stable_tpu_stack_e2e",
     schedule=SCHEDULED_TIME,
     tags=["multipod_team", "maxtext", "jax-stable-stack"],
```

Comment:
Should we add maxdiffusion and axlearn here?

Reply:
Done. Thanks